platform/darwin/src/NSExpression+MGLAdditions.h

```diff
@@ -9,6 +9,133 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NSString *MGLExpressionInterpolationMode NS_TYPED_EXTENSIBLE_ENUM;
```

Comment:
I wonder if this enumeration really should be extensible, since the interpolation modes are determined by the style specification rather than being a freeform value.

Reply:
No, this was a auto-completion problem.